package.json

```diff
@@ -30,10 +30,10 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "10.1.6",
-    "@edx/frontend-component-header": "2.4.3",
-    "@edx/frontend-platform": "1.14.9",
-    "@edx/paragon": "16.6.1",
+    "@edx/frontend-component-footer": "10.2.1",
+    "@edx/frontend-component-header": "2.4.5",
+    "@edx/frontend-platform": "1.15.1",
+    "@edx/paragon": "19.6.0",
```

Comment:
Updating all these packages here updates the `jsx` of the components coming from these packages and in turn causes the tests to fail due to mismatches between snapshot and current tree. We have to update the snapshot to latest using `npm test -- -u`. Also updati